src/components/ui/card.tsx

```
@@ -0,0 +1,92 @@
+  );
+}
+
+export {
+  Card,
+  CardHeader,
+  CardFooter,
+  CardTitle,
+  CardAction,
+  CardDescription,
+  CardContent,
+};
```

ESLint

[error] Run autofix to sort these exports!

(simple-import-sort/exports)